pkg/operatorsource/factory.go

```diff
@@ -59,11 +59,8 @@ func (s *phaseReconcilerFactory) GetPhaseReconciler(logger *log.Entry, opsrc *ma
 	case phase.OperatorSourceValidating:
 		return NewValidatingReconciler(logger, s.datastore), nil
 
-	case phase.OperatorSourceDownloading:
-		return NewDownloadingReconciler(logger, s.registryClientFactory, s.datastore, s.client, s.refresher), nil
-
 	case phase.Configuring:
-		return NewConfiguringReconciler(logger, s.datastore, s.client), nil
+		return NewConfiguringReconciler(logger, s.registryClientFactory, s.datastore, s.client, s.refresher), nil
 
 	case phase.OperatorSourcePurging:
 		return NewPurgingReconciler(logger, s.datastore, s.client), nil
```

Comment:
Higher up in this file there is a diagram that refers to the Downloading phase. We should update it to remove that reference.

Reply:
I don't see this update.